core/session.rb

```diff
@@ -499,7 +500,8 @@ def commands
     when 'install_product'
       exit_code = NodeProduct.install_product(ARGV.shift)
     when 'public_keys'
-      exit_code = publicKeys(ARGV.shift)
+      command = ConfigureNetworkCommand.new(ARGV.shift, @keyFile, @labels, $out)
```

Comment:
The list of arguments for this command must be the same as other commands. The Generate command is on the way to becoming good one :)